templates/node.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  env: {
+    node: true,
+  },
+  extends: [
+    'eslint-config-digitalbazaar/vue',
```

Comment:
this is the vue config.

Reply:
thanks for catching that omg I am horrible at this. should be corrected now.